src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -90,6 +90,25 @@ private static boolean isImportAssignment(Node statement) {
             || rightHandSide.getFirstChild().matchesQualifiedName("goog.module.get"));
   }
 
+  /** Matches `const {foo} = bar` */
```

Comment:
Does it also match `const {foo, bar} = buz`? Expand comment.

Reply:
Further comments seem to say that you also support `const {foo: bar} = buz`.

Reply:
Done.